### DISCUSSION TITLE
Inline Image Resize within -draw 'image over...' is ignored

### DISCUSSION THREAD

**Discussion by harvey-sg** [ImageMagick/ImageMagick#4195] [Help]
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am using the latest version of ImageMagick
- [x] I have searched [open](https://github.com/ImageMagick/ImageMagick/issues) and [closed](https://github.com/ImageMagick/ImageMagick/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
The inline resize seems to be ignored 
It used to work at least until version 7.0.10-17, but doesn't work with the current version
**Output with 7.0.10-17** 
![test-7 0 10-17](https://user-images.githubusercontent.com/29485120/96455617-0970c080-1250-11eb-80fd-7ef4af667b7c.png)
**Output with 7.0.10-34** 
![test-7 0 10-34](https://user-images.githubusercontent.com/29485120/96455748-3624d800-1250-11eb-82d3-d4f28a96307c.png)

### Steps to Reproduce
```
magick logo: png:- | magick -size 600x600 xc:transparent -draw 'image over 100,100 0,0 "-[100x100]"' ~/test.png
```

### System Configuration

- ImageMagick version: 7.0.10-34
- Environment (Operating system, version and so on): mac OS 10.15
- Additional information: IM built with brew

**Comment by fmw42** [comment]
Works fine for me on IM 7.0.10.34 Q16 Mac OSX Sierra.

`magick logo: png:- | magick -size 600x600 xc:transparent -draw 'image over 100,100 0,0 "-[100x100]"' yyy.png`


![yyy](https://user-images.githubusercontent.com/13458218/96475758-3e95f800-11e9-11eb-91cd-428abf30c3f7.png)

**Comment by harvey-sg** [comment]
> Works fine for me on IM 7.0.10.34 Q16 Mac OSX Sierra.
> 
> `magick logo: png:- | magick -size 600x600 xc:transparent -draw 'image over 100,100 0,0 "-[100x100]"' yyy.png`
> 
> ![yyy](https://user-images.githubusercontent.com/13458218/96475758-3e95f800-11e9-11eb-91cd-428abf30c3f7.png)

Yep, it is consistent with the above image labelled `Output with 7.0.10-34`. But shouldn't the wizard image be 100x100 at coordinate (100,100) within the 600x600-canvas ?

**Comment by fmw42** [comment]
First number in -draw are offsets and second are dimensions. See https://imagemagick.org/script/magick-vector-graphics.php

So you do not need the brackets.

`magick logo: png:- | magick -size 600x600 xc:transparent -draw 'image over 0,0 100,100 "-"' zzz.png
`

![zzz](https://user-images.githubusercontent.com/13458218/96527998-69f30400-1236-11eb-804c-72c8b61ff62f.png)

and

`magick logo: png:- | magick -size 600x600 xc:transparent -draw 'image over 400,400 100,100 "-"' zzzz.png
`

![zzzz](https://user-images.githubusercontent.com/13458218/96528370-6613b180-1237-11eb-9ff6-7120b2ef3f8f.png)

**Comment by harvey-sg** [comment]
Thanks @fmw42, the suggested syntax is not totally equal to the bracket syntax. As the bracket syntax allows to maintain image ratio as well.
Also not sure if it is something to look at,  the bracket syntax (embedded in `-draw image over`) used to work in previous IM versions.

**Comment by fmw42** [comment]
Are you sure you have the order of the offset and dimensions correct in your old example? My syntax also works if you add the brackets, because I think IM ignores that.

**Comment by harvey-sg** [comment]
Ran the same test on different IM versions (7.0.10-17 vs. 7.0.10-34), and the outcome is different. The latest IM version seems to ignore the brackets.
`magick logo: png:- | magick -size 600x600 xc:transparent -draw 'image over 100,100 0,0 "-[100x100]"' ~/test.png`
